implementations/rust/ockam/ockam_identity/src/secure_channel/encryptor_worker.rs

```diff
@@ -122,6 +122,7 @@ impl Worker for EncryptorWorker {
     }
 
     async fn shutdown(&mut self, _context: &mut Self::Context) -> Result<()> {
+        _context.stop_worker(self.addresses.decryptor_internal.clone()).await?;
```

Comment:
Let's rename `_context` to `context`. In Rust `_` prefix indicates that a variable is not used.

Reply:
Also, could you please ignore the error from this call here? There is the possibility that this call will fail. For example, if Decryptor is shutting down the Encryptor, then by the time we reach that statement, Decryptor Address no longer exists.